data/category-porn

```diff
@@ -996,6 +999,7 @@ karupsha.com
 karupspc.com
 katestube.com
 keezmovies.com
+kemono.party
```

Comment:
I think this site does not fit the porn category very well.
<img width="1358" alt="image" src="https://user-images.githubusercontent.com/32576256/172016974-2e232b4c-8479-47f5-b956-28cd30806423.png">

Reply:
> I think this site does not fit the porn category very well. <img alt="image" width="1358" src="https://user-images.githubusercontent.com/32576256/172016974-2e232b4c-8479-47f5-b956-28cd30806423.png">

Although the front page represent this website is a public archiver, you can see https://kemono.party/posts .

This website storage a lot of pornography content. Actually, this site mainly collects pornography content from the list which is written in front page.

Reply:
u r making a point.